contracts/chain-adapters/Arbitrum_L2_Forwarder.sol

```diff
@@ -0,0 +1,101 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
+
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import { ITokenMessenger } from "../external/interfaces/CCTPInterfaces.sol";
+import { CircleCCTPAdapter, CircleDomainIds } from "../libraries/CircleCCTPAdapter.sol";
+import { ArbitrumInboxLike, ArbitrumL1ERC20GatewayLike } from "../interfaces/ArbitrumBridgeInterfaces.sol";
+import { ForwarderBase } from "./ForwarderBase.sol";
+import { Arbitrum_AdapterBase } from "./Arbitrum_AdapterBase.sol";
+
+/**
+ * @notice Contract containing logic to send messages from L2 to AVM L3s.
+ * @notice This contract is for interfacing with AVM bridges that use the same
+ * native token on L3 as the native token on L2.
+ * @custom:security-contact bugs@across.to
+ */
+
+// solhint-disable-next-line contract-name-camelcase
+contract Arbitrum_L2_Forwarder is ForwarderBase, Arbitrum_AdapterBase {
```

Comment:
alternative name could be `Arbitrum_L2Adapter` for similar reasons to [here](https://github.com/across-protocol/contracts/pull/610/files#r1773668983)

Reply:
https://github.com/across-protocol/contracts/pull/610/commits/05dfc6eefae26f129ab68223a011766894ec144b